event_generation/locustfile.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+import random
+import json
+
+from locust import User, task, between
+import boto3
+
+import config as cfg
```

Comment:
respecto a como esta configurada la importacion de variables de entorno y saber cuales son las best practices. En lugar de tener un archivo (.env) en el que definirlas, para guardarlas desde el config.py y finalmente importar ese config en el codigo, no seria mejor directamente guardar las variables de entorno en un utils.py e importar directamente ese?

Reply:
What difference is there between what you suggest and what we have? just the file name right? From my perspective, it is better to use file names which are descriptive, "utils" is too abstract and could really be anything. "config" gives a better understanding of what the file is for.